### PR TITLE
Library

### PR DESCRIPTION
**Pull request by hmunfru**
### REVIEWER

@jesuspg 
### Description

Library missing
### Test

Done

### CODE REVIEW

File: package-generator/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,9 @@ RUN apt-get update && sudo apt-get -y install python-pip python-dev \
   libffi-dev zip python-mysqldb mysql-server
 RUN git clone https://github.com/telefonicaid/fiware-enablers /opt/fiware-enablers
 WORKDIR /opt/fiware-enablers/package-generator
+RUN pip install nose pytz
```
Comment (flopezag) [inline package-generator/Dockerfile:8]: Ey Warning, it was the problem that was commented today by Jesús and myself... due to a problem with the version of pip, correct me if I am wrong @jesuspg

Comment (jesuspg) [inline package-generator/Dockerfile:8]: yes, right!

Comment (jesuspg) [inline package-generator/Dockerfile:8]: If problem persist with another dependencies, please. I recommend upgrade pip

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**